boards/sparkfun/artemis_nano/artemis_nano_defconfig

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: Apache-2.0
+# Artemis Nano Kernel Configuration
+
+# Enable Console and UART
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+# Enable GPIO support
+CONFIG_GPIO=y
+
+# Enable LED support
+CONFIG_LED=y
+CONFIG_LED_GPIO=y
```

Comment:
Including drivers should be done from the application level, the basic blinky sample actually just uses the GPIO device driver directly, and `CONFIG_GPIO` is selected by the sample :)

The only drivers commonly included here are the ones required for the console, so even GPIO may be left out here :)

Reply:
Understood. Thanks